### PR TITLE
helper function to set some parities

### PR DESCRIPTION
**Pull request by Jojosito**

### CODE REVIEW

File: data/set_parities.h [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+/// \file
+
+#ifndef yap_set_parities_h
+#define yap_set_parities_h
+
+#include "ParticleTable.h"
+#include "QuantumNumbers.h"
+
+namespace yap {
```
Comment (greenwald) [inline data/set_parities.h]: I think we should not put this in the `yap` namespace.

File: include/QuantumNumbers.h [hunk 2]
```diff
@@ -82,6 +82,15 @@ class QuantumNumbers
 
     /// @}
 
+    /// \name Setters
+    /// @{
+
+    /// set Isospin * 2
```
Comment (greenwald) [inline include/QuantumNumbers.h]: correct the doxy.

I think we should also add setters for everything, if we have for some.

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -1,10 +1,11 @@
 set(INCLUDE_DIRECTORIES
+	${YAP_SOURCE_DIR}/data
```
Comment (greenwald) [inline src/CMakeLists.txt]: This shouldn't be necessary.

File: src/PDL.cxx [hunk 4]
```diff
@@ -84,6 +85,7 @@ ParticleTable read_pdl_file(const std::string& filename)
     ParticleTable particleFactory;
     std::copy(PDLIterator(inFile), PDLIterator::end(),
               inserter(particleFactory));
+    set_parities(particleFactory);
```
Comment (greenwald) [inline src/PDL.cxx]: `set_parties` is a function created for the particular `.pdl` file in our data directory. We should not use it in the main `yap` code.

**Comment by Jojosito** [comment]
Will reopen pull request with changed commits

**Pull request closed without merging**